packages/next/src/client/components/global-not-found.tsx

```diff
@@ -0,0 +1,79 @@
+const styles: Record<string, React.CSSProperties> = {
```

Comment:
This is the exact same as `error-fallback` here https://github.com/vercel/next.js/blob/canary/packages/next/src/client/components/http-access-fallback/error-fallback.tsx right?

Can we just import and re-use to keep styles and such in sync instead of duplicating?